src/js/components/accordion.js

```diff
@@ -9,24 +9,30 @@ var $ = require('jquery');
  * @param {jQuery} el A jQuery html element to turn into an accordion.
  */
 function Accordion ($el) {
-  var self = this;
-  this.$root = $el;
+  var self = this; // Node
+  this.$root = $el; // jQuery powered element
 
   // delegate click events on each <button>
+  //         v------- utils/dispatch
```

Comment:
What are these comments? Are they a special syntax for documentation or something?